File: Student.py
class Student:
    def __init__(self):
        self.id = 0
        self.firstName = ""
        self.lastName = ""
        self.startingYear = ""
        self.major = None
        self.email = ""

    def getId(self):
        return self.id

    def setId(self, id):
        self.id = id

    def getFirstName(self):
        return self.firstName

    def setFirstName(self, firstName):
        self.firstName = firstName

    def getLastName(self):
        return self.lastName

    def setLastName(self, lastName):
        self.lastName = lastName

    def getStartingYear(self):
        return self.startingYear

    def setStartingYear(self, startingYear):
        self.startingYear = startingYear

    def getMajor(self):
        return self.major

    def setMajor(self, major):
        self.major = major

    def getEmail(self):
        return self.email

    def setEmail(self, email):
        self.email = email

File: src/com/java/Student.java
package com.java;

public class Student {
    Integer id;
    String firstName;
    String lastName;
    String startingYear;
    MajorEnum major;
    String email;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public String getStartingYear() {
        return startingYear;
    }

    public void setStartingYear(String startingYear) {
        this.startingYear = startingYear;
    }

    public MajorEnum getMajor() {
        return major;
    }

    public void setMajor(MajorEnum major) {
        this.major = major;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }
}
